info.limpet.stackedcharts.ui/src/info/limpet/stackedcharts/ui/editor/parts/ChartEditPart.java

```diff
@@ -197,17 +198,40 @@ public void notifyChanged(Notification notification)
     {
       StackedchartsPackage pckg = StackedchartsPackage.eINSTANCE;
       Object feature = notification.getFeature();
```

Comment:
@dinkoivanov - I had trouble scanning through the feature types in the *if* construct.  But, I did realise that the test for PlainStyling_Color was missing.

I tried to think of an alternate strategy for the *if* construct where it was easier to scan through the features.

My implementation has inferior performance because of:
* your implementation drops out as soon as a match is found (the following equality comparisons aren't called)
* your implementation only does the **refreshChildren** test if **refreshVisuals** hasn't been called.

But, this method only gets called once per UI change, and the UI changes themselves will take many, many times longer than the performance costs described above.

If you can think of an alternate representation that is more architecturally attractive, then I welcome your thoughts.